linux/net/rina/dtcp-utils.c

```diff
@@ -346,6 +349,14 @@ static struct dtcp_config * dtcp_config_create_gfp(gfp_t flags)
           }
         */
 
+        tmp->dtcp_ps = policy_create();
```

Comment:
Both are used when dtp and dtcp are created:

 710         dtp = dtp_create(tmp->dt,
 711                          container->rmt,
 712                          tmp,
 713                          policy_name(connection->policies_params->dtp_ps),
 714                          connection);

You could delete that specific member of the connection struct but it would be an isolated policy_detroy in the efcp_create function.... Moreover, the connection_destroy function still needs to consider that if dtp_ps is present then free it. If we do a policy_destroy in efcp_create when we destroy the connection the pointer connection->dtp_ps won't be null and we will be freeing it again.